libc/config/linux/aarch64/entrypoints.txt

```diff
@@ -508,12 +508,14 @@ if(LIBC_TYPES_HAS_FLOAT16)
     libc.src.math.canonicalizef16
     libc.src.math.ceilf16
     libc.src.math.copysignf16
+    libc.src.math.f16add
```

Comment:
no `f16addl` and `f16addf128` for aarch64?

Reply:
See my comment above:

> I haven't enabled these on AArch64 because I think they will cause the same Clang 11 backend error (failure to select instructions for `fp_round` from `f128` to `f16`) as in #96952 due to this code:
> 
> https://github.com/llvm/llvm-project/pull/96787/files#diff-9916523646ef256733d16b860bb59b31d4cc2448632c871b6b2256fddadf685bR104-R107

https://github.com/llvm/llvm-project/pull/97072#discussion_r1658925698

I could still try just in case.